.github/workflows/build-zip-file.yml

```diff
@@ -18,12 +18,17 @@ jobs:
       - name: "Build the plugin"
         id: build_plugin
         uses: ./.github/actions/build
+
+      - name: "Unzip the file (prevents double zip issue)"
+        run: |
+          echo ":information_source: Unzipping the zip file to a folder. This is to prevent a double-zip issue ([known limitation](https://github.com/actions/upload-artifact#zipped-artifact-downloads)). The increase of the artefact's size is expected since we're uploading the folder instead of a zip." >> $GITHUB_STEP_SUMMARY
```

Comment:
Can we re-write `Unzipping the zip file to a folder. This is to prevent a double-zip issue` as `Unzipping to a folder to prevent double-zip issue`. Also `The increase of the artefact's size is expected` as `Increase in artifact size is expected`

Also fix the spelling - `artifact` instead of `artefact`

Reply:
Fixed